### PR TITLE
Dls 5445 mongo upgrade

### PR DESCRIPTION
**Pull request by priyankadhiman3006**
ticket details -> https://jira.tools.tax.service.gov.uk/browse/DLS-5445

### CODE REVIEW

File: app/events/BulkEvent.scala [hunk 1]
```diff
@@ -39,3 +40,30 @@ class BulkEvent(userId: String,
...
+            dualCalcs = request.calculationRequests.collect {
+                case a if a.isDualCalOne => true
+                case b if b.isDualCalZero => false
+            },
+            calcTypes = request.calculationRequests.filter(_.validCalculationRequest.isDefined)
+              .filter(_.calculationResponse.isDefined)
+              .flatMap(_.validCalculationRequest.map(_.calctype.get)))
+    }
+
+    }
```
Comment (ginnyamazed) [inline app/events/BulkEvent.scala:69]: Please add new line to end

File: app/models/BulkCalculationRequest.scala [hunk 2]
```diff
@@ -106,9 +108,9 @@ case class ProcessReadyCalculationRequest(bulkId: String,
 
 object ProcessReadyCalculationRequest {
   // $COVERAGE-OFF$
+  implicit val dateFrmat = MongoJavatimeFormats.localDateFormat
```
Comment (ginnyamazed) [inline app/models/BulkCalculationRequest.scala:111]: would it be difficult to correct this spelling error?

File: conf/application.conf [hunk 3]
```diff
@@ -157,7 +155,7 @@ microservice {
             host = localhost
             port = 2003
             prefix = play.${appName}.
-            enabled = true
+            enabled = false
```
Comment (ginnyamazed) [inline conf/application.conf:158]: why is the session timeout enabled changed to false?

File: test/actors/ProcessingSupervisorSpec.scala [hunk 4]
```diff
@@ -91,10 +91,7 @@ class ProcessingSupervisorSpec extends TestKit(ActorSystem("TestProcessingSystem
 
       when(mockRepository.findRequestsToProcess()).thenReturn(Future.successful(Some(List(processReadyCalculationRequest))))
       within(20 seconds) {
-
-        println("sending start")
         processingSupervisor ! START
```
Comment (ginnyamazed) [inline test/actors/ProcessingSupervisorSpec.scala:94]: ?

File: app/models/BulkCalculationRequest.scala (hunk 2 above)
Comment (priyankadhiman3006) [reply app/models/BulkCalculationRequest.scala:111]: nopes, i can fix that

File: conf/application.conf (hunk 3 above)
Comment (priyankadhiman3006) [reply conf/application.conf:158]: It was for my local testing, will change it back

File: test/actors/ProcessingSupervisorSpec.scala (hunk 4 above)
Comment (priyankadhiman3006) [reply test/actors/ProcessingSupervisorSpec.scala:94]: just to remove println statements

**Review by ginnyamazed — APPROVED** [review]

File: conf/application.conf (hunk 3 above)
Comment (priyankadhiman3006) [reply conf/application.conf]: PR comments addressed

**Review by ginnyamazed — APPROVED** [review]

**Pull request merged**